### ISSUE TITLE
Internet Explorer Support?

### ISSUE THREAD

**Issue by cgross** [ozone-development/ozp-iwc#397]
The IWC website lists "limited support for IE11" only but I was under the impression that the OZP Webtop supported lower versions of IE.  I'm familiar with the local-storage events suckiness with IE but my understanding is that only limited IWC communication across tabs/browser windows.  But if you iframe in widgets, the outer frame and the inner iframe can still communicate over IWC.

Is that true?  Can you guys give a clearer explanation of your IE support?

**Comment by Kevin-K** [comment]
Correct, the IE11 bug prevents Iframes of *different* tabs/windows from receiving localstorage events, which in the IWC relates to other tabs/windows not receiving IWC traffic. The investigation on this is covered in #301, and theres a gif that demonstrates the behavior (not IWC but just localstorage events) http://gifx.co/z4fn

**Comment by jziggas** [comment]
This has become a blocker for our organization. Any news since May?
